tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -913,3 +918,58 @@ func createStaticPvAndPvcInParallel(client clientset.Interface, ctx context.Cont
 
 	}
 }
+
+// triggerFullSyncInParallel triggers full sync on demand in parallel
+func triggerFullSyncInParallel(ctx context.Context, client clientset.Interface, wg *sync.WaitGroup) {
+	defer wg.Done()
+	restConfig := getRestConfigClient()
+	crd := getTriggerFullSyncCrd(ctx, client, restConfig)
+	updateTriggerFullSyncCrd(ctx, restConfig, crd)
```

Comment:
we need to wait for full sync to finish after this right

Reply:
Done